src/Pages/Home/Home.jsx

```diff
@@ -1,12 +1,10 @@
-import { Button } from "../../Components";
-
 import style from "./Home.module.scss";
+import Landing from "../../Components/Landing/Landing";
```

Comment:
Follow the template properly. Import and export the component in `src/Components/index.jsx` as 

```jsx
import Button from "./Button/Button";
import Landing from "./Landing/Landing";

export {Button, Landing};
```

and then you can use it here as 

```jsx

import { Landing } from "../../Components";
```